test/trajectory-subsetter/local_utilities.py

```diff
@@ -0,0 +1,19 @@
+""" A module containing utility functionality used by the Trajectory Subsetter
+    regression tests. These functions are kept out of the Jupyter notebook to
+    increase the readability of the regression test suite.
+
+"""
+
+from os import listdir, remove
+
+
+def remove_results_files() -> None:
+    """Remove all NetCDF-4 files downloaded during the Swath Projector
```

Comment:
HDF5 files in Trajectory Subsetter regression test? Also, could this be a shared utility or do we not really use it anywhere else yet?

Reply:
There is something similar in the #95, and from the documentation string here, I'm going to go out on a limb and suggest maybe there is a similar function in the Swath Projector test utilities.

Reply:
I commented below

> I reckon the remove_results_files could be moved into the shared utilities (maybe with an optional argument or two, such as a directory to clean up, or a file extension), but I'm not going to hold the PR up over that.

I prefer using a temporaryDirectory, so I didn't move it into the shared group. I just didn't up date the trajectory subsetter tests to use a tempdir.

Reply:
I could be convinced to move this to shared, but not for this PR. And I'd need a decent reason probably, because I'd bet they're already different.  just looked, they are different, this one has the wrong comment on it.

Reply:
I fixed the comment though: [e955ef7](https://github.com/nasa/harmony-regression-tests/pull/104/commits/e955ef74569dc3c2616150b2a5725ac71d6c2226)